cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,58 @@
+import PageObject from '../PageObject';
+
+class SettingsPageObject extends PageObject {
+  url = '/settings';
+  get usernameField() {
+    return cy.getByDataCy('username-settings');
+  }
+
+  get bioField() {
+    return cy.getByDataCy('bio-settings');
+  }
+
+  get emailField() {
+    return cy.getByDataCy('email-settings');
+  }
+
+  get passwordField() {
+    return cy.getByDataCy('password-settings');
+  }
+
+  get submitBtn() {
+    return cy.getByDataCy('submit-btn');
+  }
+
+  get logoutBtn() {
+    return cy.getByDataCy('logout-btn');
+  }
+
+  assertProfilePage(username = 'riot') {
+    return cy.url().should('contain', `/profile/${username}`);
```

Comment:
Verify that the `assertProfilePage` method correctly checks the URL for the profile page. Ensure that the username is correctly appended to the URL.